commands/search-supabase-docs/command.toml

```diff
@@ -0,0 +1,15 @@
+name = "Search Supabase Docs"
+description = "Command to search through Supabase documentation and open them in your browser."
```

Comment:
I think it's fine to just leave "Quickly search through Supabase documentation"

Reply:
Should I make the change now? (but the pr is merged so that will require another pr)

Reply:
It's published already. 

https://slapdash.com/commands/search-supabase-docs

Reply:
Oh ok